File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_SiteDatabaseConnectionsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_SiteDatabaseConnectionsService : BaseService<Sys_SiteDatabaseConnections, ISys_SiteDatabaseConnectionsRepository>
    , ISys_SiteDatabaseConnectionsService
    {
    private readonly ISys_SiteDatabaseConnectionsRepository _Sys_SiteDatabaseConnectionsRepository;
    public Sys_SiteDatabaseConnectionsService(ISys_SiteDatabaseConnectionsRepository Sys_SiteDatabaseConnectionsRepository) : base(Sys_SiteDatabaseConnectionsRepository)
    {
    _Sys_SiteDatabaseConnectionsRepository = Sys_SiteDatabaseConnectionsRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_SiteDatabaseConnections.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_SiteDatabaseConnections } from './Sys_SiteDatabaseConnections.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_SiteDatabaseConnectionsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'SiteCode', 'SiteName', 'SiteConnectionCode', 'Enabled'];

    List: Sys_SiteDatabaseConnections[] | undefined;
    ListFilter: Sys_SiteDatabaseConnections[] | undefined;
    FormData!: Sys_SiteDatabaseConnections;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_SiteDatabaseConnections";
    }
}

